src/components/Slider/index.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import SwiperCore, { Navigation } from "swiper";
+import { Swiper, SwiperSlide } from "swiper/react";
+
+import "swiper/swiper.scss";
+import "swiper/components/navigation/navigation.scss";
+
+import { SliderContainer, SliderTitle,Holder } from "./styles";
+import MovieCard from "../MovieCard";
+
+SwiperCore.use([Navigation]);
+
+export default function Slider({ title, movies, type }) {
+    const breakpoints = {
```

Comment:
Can you move `breakpoints` outside of the function since it is a constant value as to avoid being caught up with the rerendering of the components when the props change?